main.py

```diff
@@ -105,7 +116,23 @@ def main(args):
     # Create the model
     # TODO: Check and see how many latent dims should be used and insert CLI argument
     if args.use_pretrained:
-        model = tf.saved_model.load(results_dir)
+        if args.epoch_num == -1:
+            # Find last checkpoint
+            args.epoch_num = extract_epoch_num(results_dir)
+
+        model = StackGAN1(
+            img_size=dataset_dims,
+            num_latent_dims=100,
```

Comment:
Should probably be a dynamic variable in command line/config

Reply:
Am I hearing more CLI arguments? :)

Reply:
I'll ignore this because it actually isn't getting used yet